platform-hub-api/config/routes.rb

```diff
@@ -14,6 +14,9 @@
     post '/me/complete_services_onboarding', to: 'me#complete_services_onboarding'
     post '/me/global_announcements/mark_all_read', to: 'me#global_announcements_mark_all_read'
 
+    get '/feature_flags', to: 'feature_flags#index'
+    get '/feature_flags/:flag', to: 'feature_flags#show'
```

Comment:
Just to future proof, can do this as:

```
resources :feature_flags, only: [ :index, :show ]
```

That way we can easily add in the other actions for admins to manage flags (which I can help with in a later PR).